src/components/CodeView/utils.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { RefractorNode } from 'refractor';
+import makeClassName from 'classnames';
+
+const mapChild = (
+  child: RefractorNode,
+  i: number,
+  depth: number,
+): React.ReactElement | string => {
+  if (child.tagName) {
+    // @ts-ignore
+    const className = makeClassName(child.properties.className);
+
+    return React.createElement(
+      child.tagName,
+      Object.assign({ key: `code-child-${depth}-${i}` }, child.properties, {
```

Comment:
I know this is copied code but you can use spreads here to make it more readable.